metrics/network/network-nginx-ab-benchmark.sh

```diff
@@ -17,62 +17,54 @@
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 # Description:
-#  Test (host<->docker) network performance using
-#  an nginx container and running the Apache 
-#  benchmarking tool in the host to calculate the 
-#  requests per second
+# This test measures the request per second from the interconnection
+# between a server container and the host using ab tool.
+# container-server <----> host
 
 set -e
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
-
 source "${SCRIPT_PATH}/../lib/common.bash"
+TEST_NAME="network nginx ab benchmark"
 
 # Ports where it will run
-port=80:80
+port="80:80"
 # Image name
-image=nginx
+image="nginx"
 # Url
-url=localhost:80
+url="localhost:80"
 # Number of requests to perform
 # (large number to reduce standard deviation)
 requests=100000
 # Number of multiple requests to make
 # (large number to reduce standard deviation)
 concurrency=100
 
-function setup {
-	runtime_docker
-	kill_processes_before_start
-}
-
-# This script will perform all the measurements using a local setup
-
-# Test single host<->docker requests per second using nginx and ab
-
-function nginx_ab_networking {
+function nginx_ab_networking() {
+	# Verify apache benchmark
 	cmds=("ab")
 	check_cmds "${cmds[@]}"
-	setup
-	container_name="docker-nginx"
-	total_requests=$(mktemp)
+	extra_args=" -p $port"
+	total_requests="tmp.log"
+
+	# Initialize/clean environment
+	init_env
 
-	$DOCKER_EXE run -d --name ${container_name} --runtime $RUNTIME -p ${port} ${image} > /dev/null
+	# Launch nginx container
+	$DOCKER_EXE run -d -p $port $image
 	sleep_secs=2
 	echo >&2 "WARNING: sleeping for $sleep_secs seconds (see https://github.com/01org/cc-oci-runtime/issues/828)"
```

Comment:
Can you change this message for something like: "Sleeping for $sleep_secs to let the container start correctly"
Issue 828 is already closed.

Reply:
great...!!!! thanks for update